### ISSUE TITLE
UI view doesnt show last response body and header in test replay in cypress cloud

### ISSUE THREAD

**Issue by pablosmalys** [filiphric/cypress-plugin-api#130]
When I have series of api calls using cy.api command I can not see response body and header of last request. I can see request body and header only. All api calls before are complete - with response body & headers. On screenshot you can see, that I am pinned to cy.log command, so I would expect that response of last POST request will be fully visible. 

Issue occurs in safari Version 17.1 

 cypress v : 13.5.1
"cypress-plugin-api": "^2.11.1"

<img width="1470" alt="image" src="https://github.com/filiphric/cypress-plugin-api/assets/124821772/fd833ba5-4657-4cdc-aad5-42bb2ce2c499">

**Comment by alexandreedgeiq** [comment]
I'm facing the same issue mentioned above. And it's hard to debug when we have a failed test in the last request.

**Comment by martin131** [comment]
same problem

**Comment by martin131** [comment]
It looks like it is not working only in Electron, works for me in Chrome 🤔

**Comment by filiphric** [comment]
yeah the timing of Snapshots in Cypress is super hard to work with. I’ll need some time to dive into this. right now I’m implementing a very hacky solution where after a response comes back from server, it renders an element and I use `cy.get()` to assert is there, and then do the snapshot. based on this report it seems this is still not good enough.

I’ll investigate in the following days and will give you some updates

**Comment by a8trejo** [comment]
I have a branch with fixes for this because I've had this issue too and I think fixing it can bring a lot of value for my team, giving visibility to CI is always amazing value

but I can't push my branch, maybe repository restrictions @filiphric, can you help?

**Comment by a8trejo** [comment]
Created this PR @filiphric , please some 👀 
https://github.com/filiphric/cypress-plugin-api/pull/141
